subt/ros/robot/launch/k2.launch

```diff
@@ -1,96 +1,92 @@
+<?xml version="1.0"?>
 <launch>
-	<env name="DISPLAY" value=":0.0"/>
-	<!-- LOGGING -->
-	<env name="ROSCONSOLE_CONFIG_FILE"  value="$(find robot)/custom_rosconsole.conf"/>
-        <arg name="record_sensors" default="true" />
-        <!--node pkg="rosbag" type="record" name="rosbag_record" args="record -o /home/md/ros_ws/bags/ /tf /camera/color/image_raw/compressed /camera/color/camera_info /rs_t265/odom/sample /camera/depth/image_rect_raw /camera/depth/camera_info /wheel_odom  __name:=my_bag" if="$(arg record_sensors)"/-->
-        <node pkg="rosbag" type="play" name="player" output="screen" args="--pause --clock /home/pavliik/develop/ros_ws/bags/_2020-02-14-00-35-16.bag" unless="$(arg record_sensors)"/>
-	<group unless="$(arg record_sensors)">
-		<param name="use_sim_time" value="true"/>
-	</group>
-	<group if="$(arg record_sensors)">
-                <param name="use_sim_time" value="false"/>
-        </group>
+  <!-- Announces position of the point tracked by tracking camera relative to the center of the robot. -->
+  <node pkg="tf" type="static_transform_publisher" name="tracked_point_tf" args="0 0 0 0 0 0 global odom 100" />
+  <!-- Announces the position of the center of the robot relative to the point tracked by tracking camera in Osgar.
+       The tracked point name needs to match the name of the stream sent from Osgar to ROS. -->
+  <node pkg="tf" type="static_transform_publisher" name="robot_tf" args="0 0 0 0 0 0 tracking_pose robot 100" />
```

Comment:
For a flipping robot, the "robot" frame should be its center. Making it identical to the coordinate frame of the tracking camera will certainly lead to troubles.

Reply:
Hmm, there are more sources of the pose3d. My idea is the pose should be already transformed to robot center (independently to source type).

Reply:
I see. OK.